pkg/capabilities/capabilities.go

```diff
@@ -127,9 +127,20 @@ type BaseCapability interface {
 	Info(ctx context.Context) (CapabilityInfo, error)
 }
 
+type TriggerRegistrationRequest struct {
+	TriggerID string
+	Metadata  RequestMetadata
+	Config    *values.Map
+}
+
+type TriggerRegistrationResponse struct {
```

Comment:
The name `TriggerRegistrationRequest` make sense, because it only registers once per trigger, but would response be `TriggerRegistrationResponse` or `TriggerResponse`? Since it can respond multiple times.

Reply:
Good point, I think TriggerResponse makes sense 👍